src/vm/amd64/unixasmmacros.inc

```diff
@@ -52,7 +52,6 @@ C_FUNC(\Name):
 #endif
 C_FUNC(\Name):
         .cfi_startproc
-        .cfi_def_cfa_offset 8
```

Comment:
Have you verified that the unwinding works with this change? I have added this offset intentionally before and verified that the stack unwinding works correctly at every single instruction in several asm functions. As far as I remember, it didn't work without this offset. C++ compiler does also set the offset, although it sets it to 16 instead of 8 since it also accounts for the pushed `RBP` so that it doesn't have to adjust the offset twice,
```assembler
_ZN21GlobalLoaderAllocator2IdEv:        # @_ZN21GlobalLoaderAllocator2IdEv
.Lfunc_begin1:
        .loc    104 118 0               # /home/janvorli/ProjectK/src/NDP/clr/src/vm/loaderallocator.inl:118:0
        .cfi_startproc
# BB#0:
        pushq   %rbp
.Ltmp10:
        .cfi_def_cfa_offset 16
.Ltmp11:
        .cfi_offset %rbp, -16
        movq    %rsp, %rbp
```